website/docs/r/team.html.markdown

```diff
@@ -41,6 +41,7 @@ The following arguments are supported:
 * `organization` - (Required) Name of the organization.
 * `visibility` - (Optional) The visibility of the team ("secret" or "organization"). Defaults to "secret".
 * `organization_access` - (Optional) Settings for the team's [organization access](https://www.terraform.io/docs/cloud/users-teams-organizations/permissions.html#organization-level-permissions).
+* `sso_team_id` - (Optional) Unique Identifier to control team membership via SAML. Defaults to `null`
```

Comment:
It looks like we link to the docs for the `organization_access` attribute above. Should we be doing the same for the SSO team ID docs here and in the datasource docs?

Reply:
Yah, good idea, updated